packages/studio-base/src/providers/CurrentLayoutProvider/index.tsx

```diff
@@ -242,8 +242,9 @@ export default function CurrentLayoutProvider({
       }
 
       if (event.layoutId === layoutStateRef.current.selectedLayout.id) {
-        await setSelectedLayoutId(undefined);
-        enqueueSnackbar("Your active layout was deleted.", { variant: "warning" });
+        const layouts = await layoutManager.getLayouts();
+        const currLayout = layouts[0];
+        await setSelectedLayoutId(currLayout?.id ?? undefined);
```

Comment:
The `?? undefined` is redundant.
```suggestion
        await setSelectedLayoutId(layouts[0]?.id);
```